extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -219,6 +225,18 @@ describes.realWin(
             expect(isRequired).to.be.true;
           });
       });
+
+      it('respects the xssiPrefix option', async () => {
+        const remoteConfig = {
+          'consentInstanceId': 'abc',
+          'checkConsentHref': 'https://strip-prefix/',
+          'xssiPrefix': 'while(1)',
```

Comment:
I saw the poll you ran, are we going to name is `strip-prefix` instead : )

Reply:
new gh poll is here: https://github.com/ampproject/amphtml/pull/25868#discussion_r355688547